CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [unreleased]
 
 - Fix: disable React on Rails console traces in dev style guide.
+- Fix: change accessibility radio button and checkbox.
```

Comment:
Je dirais plutôt `Make radio button and checkbox accessible by keyboard` pour être plus précis.